README.md

```diff
@@ -0,0 +1,77 @@
+# bmpflash
+
+A utility for accessing SPI flash devices using the Black Magic Debug Probe.
+
+> [!NOTE]
+> This project is currently still in early stages and under heavy development.
+
+## Building from source
+
+## Installation
+
+Binary releases are not yet available and the utility must be built from
+source.
+
+## Building from source
+
+The project is configured and built using the Meson build system, you will need to create a build directory
+and then configure the build depending on what you want.
+
+```sh
+meson setup build
+```
+
+> [!NOTE]
+> Some newer versions of meson have changed the syntax of the build script which will cause errors during this step. You may need to modify the script
+as follows: Find the line in the file `meson.build` with `'b_debug=if-release'` and change it to `'debug=false'` and try rerunning the setup command.
```

Comment:
Some further debugging turned up that the option is actually `b_ndebug` - so, we'll make a commit changing the `b_debug` line to `b_ndebug=if-release` which works and is fine. Till we do that, the advice should be to correct the option name.